src/handlers/tests/status-retriever-handler.test.ts

```diff
@@ -176,19 +176,85 @@ describe('status-retriever-handler', () => {
     expect(JSON.parse(response.body)).toEqual({ intervention: accountIsNotSuspended });
   });
 
-  it('will return status 400 if the userId in the event is empty', async () => {
+  it('will return a 400 error if the request has a missing user id in path parameters', async () => {
     const invalidPathParameters = {
       proxy: '/ais/{user_id}',
-      userId: ' ',
+      userId: undefined,
     };
     const invalidTestEvent = { ...testEvent, pathParameters: invalidPathParameters };
     const response = await handle(invalidTestEvent, mockConfig);
-    expect(logger.warn).toBeCalledTimes(1);
-    expect(logger.warn).toBeCalledWith('Attribute invalid: user_id is empty.');
     expect(response.statusCode).toBe(400);
     expect(JSON.parse(response.body)).toEqual({ message: 'Invalid Request.' });
   });
 
+  it('will return a standard 200 response when there is a request with an encoded user id', async () => {
+    const suspendedRecord = {
+      pk: encodeURIComponent('testUserID'),
```

Comment:
```suggestion
pk: encodeURIComponent('test&User?ID'),
```
include chars that will be encoded